src/validation/error.messages.ts

```diff
@@ -29,6 +29,7 @@ export enum ErrorMessages {
   COUNTRY = "Select a country from the list",
   UK_COUNTRY = "Select a country",
   POSTCODE = "Enter a postcode",
+  TRUST_DATA_EMPTY = "Paste the trust information from the Excel document into the box.",
```

Comment:
No full-stops at the end of any of our other validation error messages so you may want to check this with an analyst. Up to you.

Reply:
Fair point. Removed after speaking to our analyst